platform/platform-http-proxy/pom.xml

```diff
@@ -1,142 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
After discussing with @pklinef & @stustison, we decided this platform-http-proxy bundle no longer adds any value. Anybody that needs to enable HTTP can do so via the Pax Web configuration